production/loki-mixin/dashboards/dashboard-loki-operational.json

```diff
@@ -25,61 +25,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": 68,
   "iteration": 1588704280892,
-  "links": [
```

Comment:
why are we removing the links?

Reply:
they should be populated via the common jsonnet code; technically I didn't need to remove this since we override the contents of `links` completely when calling either of these functions: https://github.com/grafana/loki/blob/main/production/loki-mixin/dashboards/dashboard-utils.libsonnet#L42-L74

this was just meant as cleanup, it can be confusing to try and wade through dashboard json when everything other than the actual queries are going to be overwritten by jsonnet code